get_started_with_membrane/get_started_with_membrane.md

```diff
@@ -96,7 +96,7 @@ end
 
 ## Our first Pipeline
 
-The pipeline is one of the basic concepts of Membrane. It's a schema of how the data packets are flowing through our application.
+The [pipeline](../glossary/glossary.md#pipeline) is one of the basic concepts of Membrane. It's a schema of how the data [packets](../glossary/glossary.md#packet) are flowing through our application.
```

Comment:
Linking to "packets" here and in a few other places might be a little bit confusing with our current definition of "packet" in the glossary - I would change the definition though instead of removing the links

Reply:
The thing is that in the glossary we are defining a packet as something inextricably connected with network - which is not applicable to that demo, where we are playing a file located on the local filesystem :D